README.md

```diff
@@ -1 +1,5 @@
-"# project-review" 
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- [ ] Good job on using GitHub inline comets, but don't forget to explain why you ask for a change. That would make the person working on the project to understand why you are asking those changes. 
You could rephrase your comment on the README file by... 
- > `Please add descriptive README file, kindly follow the Microverse template `which contains different section of a professional README. This will make sure your README is professional, and all points are included.

Reply:
Thanks for your feedback @Tadesse-Alemayehu 
I've updated the comment 

should I send a new review request?

Reply:
Good job 👍. Yes submit a new pull request pleas.